ui-components/contextmenu.md

```diff
@@ -44,12 +44,16 @@ type ContextMenuContents = string | ContextMenuItem | SeparatorMenuItemApi | Con
 type ContextMenuItem = {
   type?: 'item';
   text: string;
+  disabled?: boolean;
```

Comment:
Can we please re-order this so the more common/important properties are first? eg in the menu items docs we have text/icon together as that's the most common thing and they are also related.